### PR TITLE
Fix some href exploits

### PR DESCRIPTION
**Pull request by monster860**
No salt please

- Adds a 1 minute cooldown to the chessboard. Attempting to win within that cooldown causes the admins to get a message
- Fixes use of spaces in chessboard.
- Fixes a somewhat serious exploit with the guest pass computer that lets you give yourself any access.
- Fixes an exploit with the robotics console that lets anyone blow a borg.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Irony: the PR

File: code/game/machinery/gameboard.dm [hunk 1]
```diff
@@ -2,59 +2,60 @@
 	name = "Virtual Gameboard"
 	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "gboard_on"
-	desc = "A holographic table allowing the crew to have fun� on boring shifts! One player per board."
+	desc = "A holographic table allowing the crew to have fun� on boring shifts! One player per board."
```
Comment (tigercat2000) [inline code/game/machinery/gameboard.dm]: what

**Comment by CrAzYPiLoT-SS13** [comment]
I do not get the point of the gameboard fix. If someone is exploiting, it will be clearly visible to any observer because of the spam of the win messages. This is somewhat pointless, although it does help if no admin is observing.

**Comment by theColdflame** [comment]
a skilled human can probably beat a computer at chess in under a minute

**Comment by VampyrBytes** [comment]
> a skilled human can probably beat a computer at chess in under a minute 

The chess AI is ranked 1700-1900 - Even a grandmaster would take more than a minute to beat them

**Comment by monster860** [comment]
You can build a chessboard in a very empty place.

**Comment by CrAzYPiLoT-SS13** [comment]
I meant observer as an actual server observer, not a by-stander.

**Comment by monster860** [comment]
Use camera obscura then

**Pull request merged**